### PR TITLE
docs: Update with the new format of podvm images

### PR DESCRIPTION
**Pull request by bpradipt**
Fixes: #822

### CODE REVIEW

**Review by snir911 — APPROVED** [review]
thanks!

File: docs/consuming-prebuilt-podvm-images.md [hunk 1]
```diff
@@ -2,13 +2,20 @@
...
 
-You can find the images available at https://quay.io/organization/confidential-containers with the *podvm-[CLOUD_PROVIDER]-[DISTRO]* name pattern. For example, https://quay.io/repository/confidential-containers/podvm-libvirt-ubuntu hosts the Ubuntu images for Libvirt.
+You can find the images available at https://quay.io/organization/confidential-containers with the *podvm-[generic|vsphere]-[DISTRO]-[ARCH]* name pattern.
 
-The easiest way to extract the qcow2 file from the podvm container image is using the [`download-image.sh`](../podvm/hack/download-image.sh) script. For example, to extract the file from the *podvm-libvirt-ubuntu* image:
+For example:
+- https://quay.io/repository/confidential-containers/podvm-generic-ubuntu-amd64 hosts the Ubuntu images that can be used with all providers (except vsphere).
+- https://quay.io/repository/confidential-containers/podvm-vsphere-ubuntu-amd64 hosts the Ubuntu images that can be used with vsphere provider.
+- https://quay.io/repository/confidential-containers/podvm-generic-ubuntu-s490x hosts the Ubuntu images that can be used for s390x architecture.
```
Comment (yoheiueda) [inline docs/consuming-prebuilt-podvm-images.md:10]: `s490x` → `s390x`

**Review by stevenhorsman — APPROVED** [review]
With Yohei's suggested fix, LGTM. Thanks!

File: docs/consuming-prebuilt-podvm-images.md (hunk 1 above)
Comment (bpradipt) [reply docs/consuming-prebuilt-podvm-images.md:10]: :-)

**Pull request merged**